src/Network/Network/AzureFirewallPolicy/NatRule/NewAzureFirewallPolicyNatRuleCollectionCommand.cs

```diff
@@ -39,9 +39,9 @@ public class NewAzureFirewallPolicyNatRuleCollectionsCommand : NetworkBaseCmdlet
 
         [Parameter(
             Mandatory = false,
-            HelpMessage = "The list of network rules")]
+            HelpMessage = "The list of nat rules")]
         [ValidateNotNullOrEmpty]
-        public PSAzureFirewallPolicyNetworkRule Rule { get; set; }
+        public PSAzureFirewallPolicyNatRule[] Rule { get; set; }
```

Comment:
changing parameter type is breaking change.

I suggest keep:
public PSAzureFirewallPolicyNetworkRule Rule { get; set; }

and add:
public PSAzureFirewallPolicyNatRule[] Rule { get; set; } as new parameter

when PSAzureFirewallPolicyNetworkRule was used, convert it to PSAzureFirewallPolicyNatRule underlying.

Reply:
I understand this is a breaking change but we are fine with it because Firewall Policy is still a preview product. Please accept this change.